src/main/resources/io/jenkins/plugins/designlibrary/Dialogs/index.jelly

```diff
@@ -0,0 +1,88 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout" xmlns:s="/lib/samples" xmlns:f="/lib/form">
+  <s:sample>
+    <p class="jdl-leading-paragraph">${%description}</p>
+    <template id="demo-template" data-title="${%Modal}">
+      <div>
+        <h2 class="jenkins-dialog__subtitle">${%Behaviour}</h2>
+        ${%closeme}.
```

Comment:
the full stop is included in the message already meaning two are showing up currently. (removing from properties file would be fine also) 
```suggestion
        ${%closeme}
```